### PR TITLE
BLADE-214 fix test

### PR DESCRIPTION
**Pull request by christopherbryanboyd**

### CODE REVIEW

**Comment by christopherbryanboyd** [comment]
ci:retest

**Comment by christopherbryanboyd** [comment]
ci:tet

File: extensions/sample-profile/src/main/java/com/liferay/extensions/sample/profile/OverriddenArgs.java [hunk 1]
```diff
@@ -16,10 +16,13 @@
...
 package com.liferay.extensions.sample.profile;
 
+import com.beust.jcommander.Parameters;
+
 import com.liferay.blade.cli.command.DeployArgs;
 
 /**
  * @author Liferay
  */
+@Parameters(commandDescription = "Overriden Deploy Command", commandNames = "deploy")
```
Comment (gamerson) [inline extensions/sample-profile/src/main/java/com/liferay/extensions/sample/profile/OverriddenArgs.java]: I see, we need to re-add the parameters.  Good work.

Comment (christopherbryanboyd) [reply extensions/sample-profile/src/main/java/com/liferay/extensions/sample/profile/OverriddenArgs.java]: Thanks :)

**Pull request merged**